app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -52,4 +56,31 @@ public function store(ReportbackRequest $request)
 
         return $this->item($reportback, $code);
     }
+
+    /**
+     * Update reportbackitem(s) that already exists.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function updateReportbackItems(Request $request)
+    {
+        $items = $this->reportbackService->updateReportbackItems($request->all());
+
+        if (empty($items)) {
+            $code = 404;
+        } else {
+            $code = 201;
+        }
+
+        $meta = [];
+
+        if (count($items) === 1) {
+            $item = $items[0];
+
```

Comment:
@chloealee I don't know what is going on, what I wrote here disappeared.

I was just saying that we can simplify this by always returning it as a collection cause it looks like the repository always returns an array of reportback items even if it is only one. We can opt to change this later.